src/eeschema/eeschema_general_top_toolbar.adoc

```diff
@@ -256,30 +256,30 @@ in a few components.
 
 This methods are illustrated below.
```

Comment:
Shouldn't this sentence start with `These` instead of `This`?

Reply:
Yes, it should. My fault.